packages/compass-aggregations/src/utils/pipeline-storage.ts

```diff
@@ -1,26 +1,10 @@
-import { promises as fs } from 'fs';
-import path from 'path';
+import { z } from 'zod';
+import { UserData } from '@mongodb-js/compass-user-data';
```

Comment:
Have you considered re-exporting zod from user-data or integrating it into class constructor? You always need zod when using user-data, every package has x2 dependencies and need to maintain the version parity now, with re-export it should be easier to maintain

Reply:
added in [f153397](https://github.com/mongodb-js/compass/pull/4797/commits/f153397ef1e4b8c6d8244fa0b5ab6331991cfae2)